### PR TITLE
migrator: addition of tests

### PR DESCRIPTION
**Pull request by drjova**
Signed-off-by: Harris Tzovanakis <me@drjova.com>

### CODE REVIEW

File: tests/integration-async/migrator/test_migrator_api.py [hunk 1]
```diff
@@ -0,0 +1,329 @@
...
+
+    assert cache.llen("legacy_records") == 2
+
+    continuous_migration()
+
+    # I don't like timeouts, it's the only way to wait for this chain
+    time.sleep(10)
+
+    record_citer = InspireRecord.get_record_by_pid_value(citer_control_number, "lit")
+    record_citing = InspireRecord.get_record_by_pid_value(citing_control_number, "lit")
```
Comment (ammirate) [inline tests/integration-async/migrator/test_migrator_api.py]: if the first one is the "citer" then the second should be the "cited"

File: tests/integration-async/migrator/test_migrator_api.py [hunk 2]
```diff
@@ -0,0 +1,329 @@
...
+
+    assert record_citing.citation_count == 1
+
+    record_citer_es = InspireSearch.get_record_data_from_es(record_citer)
+    result_citer_control_number = record_citer_es["control_number"]
+
+    assert citer_control_number == result_citer_control_number
+
+    record_citing_es = InspireSearch.get_record_data_from_es(record_citing)
```
Comment (ammirate) [inline tests/integration-async/migrator/test_migrator_api.py]: same

File: tests/integration-async/migrator/test_migrator_api.py [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+    result_citing_control_number = record_citing_es["control_number"]
+
+    assert citing_control_number == result_citing_control_number
```
Comment (ammirate) [inline tests/integration-async/migrator/test_migrator_api.py]: [we could also check that the migrator indexed properly the record by checking the citation_count of ES, WDYT?

Comment (drjova) [reply tests/integration-async/migrator/test_migrator_api.py]: good point

**Comment by drjova** [comment]
@ammirate updated

**Pull request merged**